Readme.md

```diff
@@ -71,13 +71,13 @@ To ensure PLCrashReporter supports apps that use Xcode 8.3.3, it needs to be bui
     ```
 
     This will create binaries for all platforms and adds support for arm64e to PLCrashReporter-iOS. Note the environment variable `PL_ARM64E_XCODE_PATH`. Make sure to set it to your latest Xcode version that supports arm64e, currently Xcode 10.1.
-* Verify that we have a bitcode enabled framework by running `otool -l build/Release-appletv/CrashReporter.framework/Versions/A/CrashReporter | grep __LLVM`.
+* Verify that your iOS and tvOS binaries frameworks have bitcode enabled by running `otool -l build/Release-appletv/CrashReporter.framework/Versions/A/CrashReporter | grep __LLVM` (adjust the path to the binary as necessary). If you see some output, it means the binary is bitcode enabled.
```

Comment:
I would remove "frameworks" here. Not sure how far we want to go but this only checks if bitcode is enabled but it doesn't check if it's really set. Need to archive an app for that I think.